pkg/api/write.go

```diff
@@ -148,6 +158,96 @@ func validateWriteHeaders(w http.ResponseWriter, r *http.Request) bool {
 }
 
 func buildWriteError(w http.ResponseWriter, err string) {
-	log.Error("msg", "Write header validation error", "err", err)
+	buildError(w, err, "Write header validation error")
+}
+
+func buildError(w http.ResponseWriter, err string, msg string) {
+	log.Error("msg", msg, "err", err)
 	http.Error(w, err, http.StatusBadRequest)
 }
+
+func buildImportError(w http.ResponseWriter, err string) {
+	buildError(w, err, "Import header validation error")
+}
+
+type importPayload struct {
+	Metric     map[string]string `json:"metric"`
```

Comment:
is this to insert the same values to multiple metrics, or does it store the labels key-value mappings, or something else?

Reply:
Looking at the code, it seems to be the label key-value mappings. Can we name this `labels` or `series`?